src/testCase.js

```diff
@@ -86,9 +86,14 @@ define([
 
 		// All test classes are singletons.
 		// TODO: Refactor with a super call when we"ll have super on class-side.
-		that.subclass = (function(superSubclass) {
+		that.subclass = (function(superSubclass, options) {
 			return function(builder) {
 				var klass = superSubclass.apply(that, [builder]);
+
+				if (options && options.isAbstract) {
+					return klass;
+				}
```

Comment:
@BenjaminVanRyseghem Can you explain this change? Abstract classes could get instantiated otherwise?